unittests/TestUtils/TestConfig.h

```diff
@@ -2,13 +2,16 @@
 #define UNITTEST_TESTUTILS_TESTCONFIG_H_
 
 #include "config.h"
+#include "phasar/Config/Configuration.h"
 
 #include <string>
 
 namespace psr::unittest {
 
-inline const std::string PathToLLTestFiles(PHASAR_BUILD_DIR
-                                           "/test/llvm_test_code/");
+inline PSR_CONST_CONSTEXPR std::string
```

Comment:
Let's stick to `inline const` to avoid cluttering the code base with yet another preprocessor macro.